index.bs

```diff
@@ -855,6 +863,20 @@ these steps.
    requested but not satisfied, then the value is `"default"`
    The getter steps are to return [=this=]'s {{[[CongestionControl]]}}.
 
+: <dfn for="WebTransport" attribute>rateControlFeedbackMinInterval</dfn>
+:: The application's preference, if changed from the default,
+   for the minimum interval between any two times when
+   [=this=]'s onratecontrolfeedback is fired.
+   This effectively allows the application
+   to control the maximum frequency of this event.
+
+: <dfn for="WebTransport" attribute>onratecontrolfeedback</dfn>
+:: Fired when [=this=]'s rateControlFeedback changes.
+   May be delayed by  [=this=]'s rateControlFeedbackMinInterval.
```

Comment:
Spec should define the `ratecontrolfeedback` event, not just the `onratecontrolfeedback` event handler which are separate things.

Reply:
I think that's the case now.